performance-tests/src/main/jmeter/disable-job-output.jmx

```diff
@@ -0,0 +1,340 @@
+            <stringProp name="Argument.name">host</stringProp>
+            <stringProp name="Argument.value">${__P(host, localhost)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="username" elementType="Argument">
+            <stringProp name="Argument.name">username</stringProp>
+            <stringProp name="Argument.value">${__P(username, hub-developer)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="password" elementType="Argument">
+            <stringProp name="Argument.name">password</stringProp>
+            <stringProp name="Argument.value">${__P(password, password)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="flowName" elementType="Argument">
+            <stringProp name="Argument.name">flowName</stringProp>
+            <stringProp name="Argument.value">jmeterCustomFlow</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="sourceCollection" elementType="Argument">
+            <stringProp name="Argument.name">sourceCollection</stringProp>
+            <stringProp name="Argument.value">jmeter-source-documents</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="sourceDocumentCount" elementType="Argument">
+            <stringProp name="Argument.name">sourceDocumentCount</stringProp>
+            <stringProp name="Argument.value">${__P(sourceDocumentCount, 10000)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="iterations" elementType="Argument">
+            <stringProp name="Argument.name">iterations</stringProp>
+            <stringProp name="Argument.value">${__P(iterations, 1)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="port" elementType="Argument">
+            <stringProp name="Argument.name">port</stringProp>
+            <stringProp name="Argument.value">8011</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="disableJobOutput" elementType="Argument">
+            <stringProp name="Argument.name">disableJobOutput</stringProp>
+            <stringProp name="Argument.value">${__P(disableJobOutput, false)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="authUsername" elementType="Argument">
+            <stringProp name="Argument.name">authUsername</stringProp>
+            <stringProp name="Argument.value">${__P(authUsername, admin)}</stringProp>
```

Comment:
Use test-data-hub-developer and password here; we know that user exists because it's created via "./gradlew bootstrap".